public/static/docs/command-reference/get.md

```diff
@@ -3,8 +3,7 @@
 Download a file or directory tracked by DVC or by Git into the current working
 directory.
 
-> Unlike `dvc import`, this command does not track the downloaded files (does
-> not create a DVC-file).
+> See also our `dvc.api.open()` Python API function.
```

Comment:
and dvc.api.read?

Reply:
I don't know. Do you mean link to `dvc.api.read()` from `dvc get` and to `dvc.api.open()` from `dvc import`? I don't see a direct analogy among these pairs. I think the main API function to download files is open(). read() just wraps it for easy use. Neither API fn creates an import stage.

I added a link to read() from the open() ref description now, though.